packages/discovery-react-components/src/components/DocumentPreview/components/PdfViewer/PdfViewer.tsx

```diff
@@ -1,6 +1,8 @@
 import React, { FC, useEffect, useRef, useCallback, useMemo, useState } from 'react';
 import cx from 'classnames';
-import PdfjsLib, { PDFDocumentProxy, PDFPageProxy, PDFPromise, PDFRenderTask } from 'pdfjs-dist';
+import * as PdfjsLib from 'pdfjs-dist';
```

Comment:
Seems that we only use var `PdfjsLib` for `getDocument()` and `GlobalWorkerOptions`. Why not just bring those in, instead of `import *`?
```suggestion
import { getDocument, GlobalWorkerOptions } from 'pdfjs-dist';
```